README.md

```diff
@@ -111,14 +111,54 @@ Check out how in the INSTALL section.
 
 ## Installation
 
+In all the commands listed below:
+- `example.com` is your domain name
+- `auth.example.com` is the subdomain you want to use for acme-dns
+- `198.51.100.1` is the **public** IP address of the system running acme-dns  
```

Comment:
I feel that in examples it would be better to use address in a private address space defined by [RFC 1918](https://tools.ietf.org/html/rfc1918)

Reply:
`198.51.100.1` is actually a reserved IP address for documentation purposes: https://tools.ietf.org/html/rfc5737

I'm in favor of keeping it, rather than using a private address.

Reply:
Yeah, @Daniel15 cleared that up in the comment, and I learned something. I'm in favor of keeping this as well.

Reply:
Ah, didn't see that.